cdhweb/people/tests/test_pages.py

```diff
@@ -63,6 +63,7 @@ def test_delete_handler(self, people_landing_page):
         person.delete()
         assert Profile.objects.count() == 0
 
+    @pytest.mark.skip("Skipped after blog tests cleanup - blog integration test")
```

Comment:
I am suspicious of this change... The test should either be removed or fixed, not skipped; you've removed a fixture used for the test, so it would never work again the way it did previously. (And would be harder to figure out later when the fixture is gone but no the test.)

However, I am puzzled why it would be working but not impacted by the code you've removed. I looked briefly at the test fixture and I think maybe it's old code that's no longer used by the running application - but that feels like a bigger change than we intended.